### ISSUE TITLE
Conduct test: Update error message design for the test card

### ISSUE THREAD

**Issue by kenieh** [CDCgov/prime-simplereport#3900]
### Background
With ticket #3792, we've updated the way error messages are handled to be more streamlined. Instead of an error message toaster, we're using the inline error message capabilities of the test card. 

### Design

BEFORE
![Screen Shot 2022-06-16 at 2.57.14 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/dc7bebdf-2e09-4c22-9bbd-5cfe68a53e04)

AFTER
![Screen Shot 2022-06-16 at 4.31.03 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/fbe8e523-2bb6-4e47-8f79-4ba18b2634b4)

SPEC
![Screen Shot 2022-06-16 at 2.43.25 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/dec5d66a-b28d-4a47-ad3a-336a979f8877)

### Acceptance Criteria
What needs to happen for this ticket to be complete?

### Additional context
Design should pair with an Engineer to understand all the error states we currently have for the test card.

**Comment by emmastephenson** [comment]
Would be worthwhile to do an exercise to find all the issues/errors we know could come up in a test card. 

So far, we know:
- Test date in the future

Team to do a micro bug bash on this next week during G&E sync.

**Comment by emmastephenson** [comment]
Still an issue

**Comment by fzhao99** [comment]
> Would be worthwhile to do an exercise to find all the issues/errors we know could come up in a test card.
> 
> So far, we know:
> 
> * Test date in the future
> 
> Team to do a micro bug bash on this next week during G&E sync.

@emmastephenson did we ever do this for the error states of the card? If so, are they living somewhere?

**Comment by emmastephenson** [comment]
@fzhao99 I don't believe that we did! Off the top of my head I know the future dates and the "something went wrong" error that gets thrown if there's an error on the backend, but not sure about other errors.

**Comment by lg-king** [comment]
There's also this ticket - #3165 - which design are we going with?
